nixos/lib/test-driver/test-driver.py

```diff
@@ -216,7 +216,7 @@ def __init__(self, args: Dict[str, Any]) -> None:
             if cmd:
                 match = re.search("run-(.+)-vm$", cmd)
                 if match:
-                    self.name = match.group(1)
+                    self.name = match.group(1).replace(".", "_")
```

Comment:
This file is named after the hostname, but later on this Python code uses the `machine.name` value as a Python variable name.

Reply:
I guess the problem doesn't just apply to `.`'s but all kinds of allowed characters? e.g. `-`?

Reply:
True, though I wasn't sure just how far I should go. I could add `-` too. Should I do more?